news/2020-03-23-community-mentorship-program-spring-2020-signups-now-open.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Community Mentorship Program Spring 2020 Signups Now Open
+date: 2020-03-23 10:00:00 +0000
+---
+
+The 15th cycle of the Community Mentorship program is on its way! If you've ever found yourself wanting to expand your current knowledge and skills or help beginning mappers, this program will serve as a great opportunity for you!
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills underneath the guidance of more experienced mappers and modders in the community. Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like mapping contests), hang out with fellow mentees, and more!
+
+If you're experienced in mapping and have a desire to take those who are new under your wing, then give joining us as Mentor a shot! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1038431), you'll be able to apply right away. If you're only just getting into mapping or are eager to hone your skills, then signing up as Mentee might just be your way to do it!
+
+---
+
+For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):
+
+- Mentor signups will open on 2020-03-23 and close on 2020-03-30.
+- Mentee signups will open on 2020-04-19 and close on 2020-04-26.
+- The 2020 Spring cycle will run from 2020-04-27 to 2020-07-27.
+
+Throughout this active cycle time, the organisation team will host various events on the Mentorship Discord server, engaging and offering events for mentees and mentors to participate in. These include feedback lobbies, mapping contests, and more! Some of our previous puzzles can be found here at this [thread](https://osu.ppy.sh/community/forums/topics/565382), and previous discussions can be found [here](https://osu.ppy.sh/community/forums/topics/514122).
+
+If any of this interests you or anyone you know, please go to our [forum announcement and rules thread](https://osu.ppy.sh/community/forums/topics/1038431) for more information.
+
+We're looking forward to the future mappers of tomorrow!
```

Comment:
apparently I've commented on this before https://github.com/ppy/osu-wiki/pull/2503#discussion_r321997559

I think the part that gets me is having both "future" and "tomorrow", "future" doesn't add much

Reply:
changed to 'looking forward to the mappers and modders of tomorrow!'